core/src/main/scala/ai/h2o/sparkling/backend/external/RestApiUtils.scala

```diff
@@ -95,6 +99,26 @@ trait RestApiUtils extends RestCommunication {
   def getCloudInfoFromNode(endpoint: URI, conf: H2OConf): CloudV3 = {
     query[CloudV3](endpoint, "/3/Cloud", conf)
   }
+
+  def updateAsJson(suffix: String,
+                   params: Map[String, Any] = Map.empty,
+                   encodeParamsAsJson: Boolean = false
+                  ): String = {
```

Comment:
nit: Can be joined with the previous line.

Reply:
👍